Ui update

tescher opened the pull request:
Got all tests running, changed UI to match proposal from the State, abstracted a few things.

Issues: #2, #5, #15, #16 

NOTE: Change in DB schema, need to drop "queued" table and re-run utils/createQueuedTable.js

NigelKibodeaux commented on README.md:
```diff
@@ -63,3 +64,13 @@ Finally, you'll want to setup scheduler to run the various tasks each day. Here'
+
+## Running Tests
+
+Test are run with Mocha. To run from the command line, you'll first need to install Mocha globally:
```
this is not actually true. if you run `npm test`, it should work

tescher commented:
Fixed, also cleaned up env usage in test files so it should run mostly
without you having to set up env vars (except for db url maybe).

Tim Escher
tim@eschers.com
608-432-5799

On Mon, Sep 19, 2016 at 9:20 PM, Nigel Kibodeaux <notifications@github.com>
wrote:

> *@NigelKibodeaux* commented on this pull request.
> ------------------------------
>
> In README.md
> <https://github.com/codeforanchorage/courtbot/pull/17#pullrequestreview-681494>
> :
>
> > @@ -63,3 +64,13 @@ Finally, you'll want to setup scheduler to run the various tasks each day. Here'
>  * node runners/load.js
>  * node runners/sendQueued.js
>  * node runners/sendReminders.js
> +
> +## Running Tests
> +
> +Test are run with Mocha. To run from the command line, you'll first need to install Mocha globally:
>
> this is not actually true. if you run npm test, it should work
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/codeforanchorage/courtbot/pull/17#pullrequestreview-681494>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACcF3BAYJAo2b1JTwTxGTacs__Wzw2o0ks5qr20agaJpZM4KAwWS>
> .[image: Web Bug from
> https://github.com/notifications/beacon/ACcF3Gs7CA_6d2qGafIO05WLDHWaaE4Eks5qr20agaJpZM4KAwWS.gif]
> {"api_version":"1.0","publisher":{"api_key":"
> 05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":
> {"external_key":"github/codeforanchorage/courtbot","
> title":"codeforanchorage/courtbot","subtitle":"GitHub
> repository","main_image_url":"https://cloud.githubusercontent.com/assets/
> 143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","
> avatar_image_url":"https://cloud.githubusercontent.com/
> assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png
> ","action":{"name":"Open in GitHub","url":"https://github.
> com/codeforanchorage/courtbot"}},"updates":{"snippets":[{"
> icon":"PERSON","message":"@NigelKibodeaux reviewed
> #17"}],"action":{"name":"View Pull Request","url":"https://
> github.com/codeforanchorage/courtbot/pull/17#pullrequestreview-681494"}}}
